luigi/contrib/redshift.py

```diff
@@ -52,7 +138,7 @@ class RedshiftTarget(postgres.PostgresTarget):
     use_db_timestamps = False
 
 
-class S3CopyToTable(rdbms.CopyToTable):
+class S3CopyToTable(rdbms.CopyToTable, _CredentialsMixin):
```

Comment:
Sounds doable. Might take me a little bit though, as I have to focus on some other stuff right now.